README.md

```diff
@@ -1,42 +1,69 @@
-# What's inside
+# Containerized Ansible: What's inside #
 
-This container provide ansible toolstack inside a container.
+This container provides the ansible toolstack inside a container.
 * '''Dockerfile''' with the definition of the ansible container
-** based on OpenSUSE Tumbleweed
-** installs ansible and some additional tools
-* System Setup
-** Make sure needed packages installed on the host
----
-# sudo transactional-update pkg install podman python310-rpm
----
-*** Install ansible commands wrapers
-If current user is root /usr/local/bin is updated, for non-root users
-the user's ~/bin area is updated.
----
-as root:
-# (cd ~; podman container runlabel root-install ansible)
-as non-root
-# (cd ~; podman container runlabel user-install ansible)
----
-*** Ansible Commands
----
-ansible
-ansible-community
-ansible-config
-ansible-connection
-ansible-console
-ansible-doc
-ansible-galaxy
-ansible-inventory
-ansible-playbook
-ansible-pull
-ansible-vault
----
-
-*** Uninstall ansible commands wrapers
----
-as root:
-# (cd ~; podman container runlabel root-unstall ansible)
-as non-root
-# (cd ~; podman container runlabel user-unstall ansible)
----
+  * based on OpenSUSE Tumbleweed
+  * installs ansible and some additional tools
+## System Setup ##
+* Make sure needed packages installed on the host
+  * sudo transactional-update pkg install podman python310-rpm
```

Comment:
podman is part of the ALP Base - no need to install it

We probably should have this as a list of packages, with corresponding annotations as to what in Ansible depends on the relevant package, e.g. the `package_facts` module depends on this being installed on RPM package bases systems...

Reply:
I know, but since it is required I wanted  to have it in case someone else used this as basis on a different system.
Not sure I understand the last part python310-rpm is needed so ansible can querry what is installed.

Reply:
For target system, which is a container host OS that comes with podman already installed, explicitly saying to install podman is a redundant operation. However if someone wanted to use this container on a different platform, yes they would need to install podman, but the command to do so likely wouldn't be using transactional-update.